source/DonutNode.cpp

```diff
@@ -7,22 +7,48 @@
 using namespace cugl;
 
 /** The radius of the ship. Also the y coordinate of the center of the ship */
-constexpr unsigned int RADIUS = 580;
+constexpr unsigned int RADIUS_OFFSET = 30;
+
+/** Position to place node offscreen. */
+constexpr float OFF_SCREEN_POS = 1500;
```

Comment:
Is this actually offscreen on all screen sizes? Is it not possible to just set `visible` to be `false` on everything off screen?

Reply:
Yes, this would be off screen because our working screen width is 1024. I've tried with visibility settings but it would no longer call `draw()` if something is not visible, and SG root would have to manually set it to visible again, which would make the code too bloated.